sections/layer2/deposit/InfoBox/InfoBox.tsx

```diff
@@ -21,10 +20,22 @@ import {
 	Header,
 	StyledTable,
 } from 'sections/escrow/components/common';
+import { useRecoilValue } from 'recoil';
+import { walletAddressState } from 'store/wallet';
+import useSynthetixQueries, { DepositHistory } from '@synthetixio/queries';
 
 const InfoBox = () => {
 	const { t } = useTranslation();
-	const depositsDataQuery = useGetDepositsDataQuery();
+
+	const walletAddress = useRecoilValue(walletAddressState);
+
+	const { useGetBridgeDataQuery } = useSynthetixQueries();
+
+	const depositsDataQuery = useGetBridgeDataQuery(
+		process.env.NEXT_PUBLIC_INFURA_PROJECT_ID!,
```

Comment:
Hmmm can we maybe put all the process.env variables into a constants file? I thought it was the case

Reply:
actually, the `process.env` values in vercel come from a local `.env.local` file, so basically works as expected.

this is the pattern used to get the onboard ID as well, and matches behavior of the app prior to this PR